src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -54,13 +58,13 @@ abstract class Email(
   prisonerLastName: String,
   prisonerNumber: String,
   date: LocalDate = LocalDate.now(),
-  comments: String? = "",
+  comments: String? = null,
 ) {
   private val common = mapOf(
     "date" to date.toMediumFormatStyle(),
     "prisonerName" to prisonerFirstName.plus(" $prisonerLastName"),
     "offenderNo" to prisonerNumber,
-    "comments" to comments,
+    "comments" to (comments ?: "None entered"),
```

Comment:
This address the null comments in email issue spotted during integration with the UI and API.